astropy/utils/misc.py

```diff
@@ -815,7 +818,25 @@ def __init__(cls, cls_name, bases, members):
             instances = OrderedDict((key, value) for value, key in instances)
             setattr(cls, descriptor_cls._class_attribute_, instances)
 
-        super().__init__(cls_name, bases, members)
+        super(OrderedDescriptorContainer, cls).__init__(cls_name, bases,
+                                                        members)
+
+
+def get_parameters(members):
```

Comment:
Added docstring. It was placed here since there was generic code corresponding to OrderedDescriptors that is located in this module. Currently the code is used only in
modeling, but it fairly generic. It can be moved into modeling if necessary.